src/store/PriceLineStore.ts

```diff
@@ -23,29 +23,47 @@ export default class PriceLineStore {
     opacityOnOverlap = 0;
     showOffscreenArrows = false;
     _relative = false;
-    @observable draggable = true;
-    @observable isDragging = false;
-    @observable visible = true;
+    draggable = true;
+    isDragging = false;
+    visible = true;
     // @observable top = 0;
```

Comment:
I think we can remove these old comments

Reply:
done